source/upgrade.txt

```diff
@@ -159,6 +159,13 @@ The v4.0 driver introduces the following breaking changes to the API:
   library equivalents. For context, see the :ref:`Build System Breaking Changes
   <version-4.0-build-system-breaking>` section.
 
+- Removes the following headers:
```

Comment:
Q: would C++ devs be familiar with the term "headers" or do you think some more explanation could be useful here?

Reply:
I changed to "header files" to be more specific, but both headers/header files are mentioned throughout the docs